lib/tests/fixtures.hpp.in

```diff
@@ -3,6 +3,9 @@
 #include <string>
 #include <functional>
 
+#define JAVA_EXECUTABLE "@Java_JAVA_EXECUTABLE@"
```

Comment:
I get the same thing trying to build on a fresh Ubuntu 14.04 (Docker) image I just setup, following the README instructions.